source/macos/AdaptiveCards/AdaptiveCards/Renderers/FactSetRenderer.swift

```diff
@@ -29,45 +29,51 @@ class FactSetRenderer: NSObject, BaseCardElementRendererProtocol {
         var requiredWidth: CGFloat = 0
         // Main loop to iterate over Array of facts
         for fact in factArray {
-            let markdownParserResult = BridgeTextUtils.processText(from: fact, hostConfig: hostConfig)
-            let markdownString = TextUtils.getMarkdownString(for: rootView, with: markdownParserResult)
-            let attributedContent = TextUtils.addFontProperties(attributedString: markdownString, textProperties: BridgeTextUtils.convertFact(toRichTextElementProperties: fact), hostConfig: hostConfig)
+            let valueViewMarkdownParserResult = BridgeTextUtils.processText(from: fact, hostConfig: hostConfig, isTitleField: false)
+            let valueViewMarkdownString = TextUtils.getMarkdownString(for: rootView, with: valueViewMarkdownParserResult)
+            let valueViewAttributedContent = TextUtils.addFontProperties(attributedString: valueViewMarkdownString, textProperties: BridgeTextUtils.convertFact(toRichTextElementProperties: fact), hostConfig: hostConfig)
+            
+            let titleViewMarkdownParserResult = BridgeTextUtils.processText(from: fact, hostConfig: hostConfig, isTitleField: true)
+            let titleViewMarkdownString = TextUtils.getMarkdownString(for: rootView, with: titleViewMarkdownParserResult)
+            let titleViewAttributedContent = TextUtils.addFontProperties(attributedString: titleViewMarkdownString, textProperties: BridgeTextUtils.convertFact(toRichTextElementProperties: fact), hostConfig: hostConfig)
                          
-            let titleView = ACRFactTextField()
-            let valueView = ACRFactTextField()
-            titleView.plainTextValue = fact.getTitle()
-            titleView.setupTitleString(with: hostConfig.getFontSize(.default, size: .default))
+            let titleView = ACRTextView()
+            let valueView = ACRTextView()
+            
             // If not markdown use plain text
-            if !markdownParserResult.isHTML {
-                valueView.plainTextValue = fact.getValue()
+            if !valueViewMarkdownParserResult.isHTML {
+                valueView.string = fact.getValue() ?? ""
+            } else {
+                valueView.textStorage?.setAttributedString(valueViewAttributedContent)
+            }
+            
+            if !titleViewMarkdownParserResult.isHTML {
+                titleView.string = fact.getTitle() ?? ""
             } else {
-                valueView.attributedTextValue = attributedContent
+                titleView.textStorage?.setAttributedString(titleViewAttributedContent)
             }
             
+            titleView.customizeForFactSetCell(isTitleView: true, config: config)
+            valueView.customizeForFactSetCell(isTitleView: false, config: config)
```

Comment:
Maybe you can create a `convenience init(asFactTextFieldWith config: RenderConfig, isTitle: Bool)` in `ACRTextView` and do this is init only.